price-feeder/Makefile

```diff
@@ -22,7 +23,8 @@ endif
 ###############################################################################
 
 ldflags = -X github.com/umee-network/umee/price-feeder/cmd.Version=$(VERSION) \
```

Comment:
Looks like above we have this line: 
```
  ifeq (,$(VERSION))
    VERSION := $(BRANCH)-$(COMMIT)
  endif
```

Which resolves to `-hash` when built using this makefile, because `BRANCH` is not filled out here.

Could you add the branch to the top of this file much how `umee/makefile` does it? that way it'll look like `branch-hash` 😄

Reply:
done!